scripts/tags/channel.md

```diff
@@ -12,41 +12,70 @@ dateCreated: 2022-02-16T06:15:22.397Z
 
 ## `{channel.mention;channel}`
 
+Returns a string with a channel mention.
+
 ## `{channel.id;channel}`
 
+Returns the ID of a channel.
+
 ## `{channel.type;channel}`
 
+Returns an integer for the [channel type](https://discordpy.readthedocs.io/en/stable/api.html#discord.ChannelType):
+
+| Number | Type        | Description                                           |
```

Comment:
Remove this table as it's duplicating what the Discord documentation already documents and lets be honest no one will remember to update this list once Discord updates theirs. We follow their enum names 1:1 and there are additional validation layers which will match inconsistencies, so it should be fine.

For example, `guildtext`, `guild_text`, `GUILD_TEXT` and `GuildText` should all resolve to the same channel. Enum values are also supported, `0` will resolve to GUILD_TEXT but in general using enum names is recommended.